src/pages/workspaces/workspace/Dashboard/Dashboard.ts

```diff
@@ -104,365 +66,82 @@ interface WorkspaceDashboardProps {
-    () => getLocalPref(persistenceId)?.authDomainPanelOpen || false
-  );
-  const [tagsPanelOpen, setTagsPanelOpen] = useState(() => getLocalPref(persistenceId)?.tagsPanelOpen || false);
-  const [notificationsPanelOpen, setNotificationsPanelOpen] = useState(
-    () => getLocalPref(persistenceId)?.notificationsPanelOpen || false
-  );
-
-  useEffect(() => {
-    setLocalPref(persistenceId, {
-      workspaceInfoPanelOpen,
-      cloudInfoPanelOpen,
-      ownersPanelOpen,
-      authDomainPanelOpen,
-      tagsPanelOpen,
-      notificationsPanelOpen,
-    });
-  }, [
-    persistenceId,
-    workspaceInfoPanelOpen,
-    cloudInfoPanelOpen,
-    ownersPanelOpen,
-    authDomainPanelOpen,
-    tagsPanelOpen,
-    notificationsPanelOpen,
-  ]);
-
-  const loadWsTags = withErrorReporting('Error loading workspace tags', async () => {
-    setTagsList(await Ajax(signal).Workspaces.workspace(namespace, name).getTags());
-  });
-
-  const addTag = _.flow(
-    withErrorReporting('Error adding tag'),
-    withBusyState(setBusy)
-  )(async (tag) => {
-    setTagsList(await Ajax().Workspaces.workspace(namespace, name).addTag(tag));
-  });
-
-  const deleteTag = _.flow(
-    withErrorReporting('Error removing tag'),
-    withBusyState(setBusy)
-  )(async (tag) => {
-    setTagsList(await Ajax().Workspaces.workspace(namespace, name).deleteTag(tag));
-  });
-
-  const loadAcl = withErrorReporting('Error loading ACL', async () => {
-    const { acl } = await Ajax(signal).Workspaces.workspace(namespace, name).getAcl();
-    setAcl(acl);
-  });
-
-  const save = withBusyState(setSaving, async () => {
-    try {
-      await Ajax().Workspaces.workspace(namespace, name).shallowMergeNewAttributes({ description: editDescription });
-      await refreshWorkspace();
-    } catch (error) {
-      reportError('Error saving workspace', error);
-    } finally {
-      setEditDescription(undefined);
-    }
-  });
-
-  // Lifecycle
-  useOnMount(() => {
-    refresh();
-  });
-
-  // Render
-  const isEditing = _.isString(editDescription);
-  const brand = getEnabledBrand();
-  // @ts-expect-error
-  const { value: canEdit, message: editErrorMessage } = canEditWorkspace(workspace);
-
-  return div({ style: { flex: 1, display: 'flex' } }, [
-    div({ style: Style.dashboard.leftBox }, [
-      div({ style: Style.dashboard.header }, [
-        'About the workspace',
-        !isEditing &&
-          h(
-            Link,
-            {
-              style: { marginLeft: '0.5rem' },
-              disabled: !canEdit,
-              tooltip: canEdit ? 'Edit description' : editErrorMessage,
-              onClick: () => setEditDescription(description?.toString()),
-            },
-            [icon('edit')]
-          ),
+export const WorkspaceDashboard = forwardRef(
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  (props: WorkspaceDashboardProps, ref: React.ForwardedRef<unknown>): ReactNode => {
```

Comment:
The ref is still here to avoid a console warning.
The workspace container is very guarded about using it, so it's not a problem to not set it, and we don't need the functionality. But we do still need to wrap the component in a `forwardRef` for compatibility with `wrapWorkspace` and `WorkspaceContainer`.